.infrastructure/mysql/mysql-secret.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-secret
+  namespace: mysql
+type: Opaque
+data:
+    MYSQL_ROOT_PASSWORD: "cm9vdA==" #Base64 "root"
+    MYSQL_DATABASE: "YXBwX2Ri" #Base64 "app_db"
+    MYSQL_USER: "bXlzcWx1c2Vy" #Base64 "mysqluser"
+    MYSQL_PASSWORD: "cm9vdA==" #Base64 "root"
```

Comment:
Using 'root' as the password for MYSQL_PASSWORD is not secure. Consider using a different, stronger password for the MySQL user.